packages/vuetify/src/components/VCalendar/__tests__/VCalendarCategory.spec.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  mount,
+  Wrapper,
+  MountOptions,
+} from '@vue/test-utils'
+import { ExtractVue } from '../../../util/mixins'
+import VCalendarCategory from '../VCalendarCategory'
+
+describe('VCalendarCategory', () => {
+  type Instance = ExtractVue<typeof VCalendarCategory>
+  let mountFunction: (options?: MountOptions<Instance>) => Wrapper<Instance>
+  beforeEach(() => {
+    mountFunction = (options?: MountOptions<Instance>) => {
+      return mount(VCalendarCategory, {
+        ...options,
+        mocks: {
+          $vuetify: {
+            lang: {
+              current: 'en-US',
+            },
+          },
+        },
+      })
+    }
+  })
+
+  it('should render correctly without shortIntervals prop', async () => {
```

Comment:
this does not use the new `categoryText` prop, so I'm unsure what this is testing exactly